### DISCUSSION TITLE
Guess the charater game list problem, python discord bot

### DISCUSSION THREAD

**Discussion by seraphyy** [Rapptz/discord.py#6420] [Q&A] [answered]
.``

**Comment by Harmon758** [comment, marked as answer]
As the error says:
> `In embed.image.url: Scheme "deku" is not supported. Scheme must be one of ('http', 'https').`

`chosen_image` can randomly be chosen as "Deku" from `bnhaLinks`, and then it's set as the embed's image URL, even though it's not a proper URL.
